pkg/operation/botanist/worker.go

```diff
@@ -83,6 +83,15 @@ func (b *Botanist) DeployWorker(ctx context.Context) error {
 			}
 		}
 
+		if worker.CRI != nil {
```

Comment:
Just for me to understand: Can you quickly explain why we add these labels?

Reply:
Yes, in the container runtime extensions the installation is done by daemon set only for the workers (nodes) who has those annotations.

Reply:
Understood, makes sense. However, where are the nodes labelled accordingly so that the `DaemonSet`s can define a proper label selector?

Reply:
Well, i assumed all labels set here to the worker will also be set to the node.
Actually it is different from what i did in the POC (Manipulated the Kubelet flags so the nodes where labelled. I was still planning to check this.
If it doesn't work like that i will revert it to the _Kubelet

Reply:
OK, no, it doesn't work like that. In the PR we discussed that the node labels can be done via the [kubelet flags](https://github.com/gardener/gardener/blob/master/charts/seed-operatingsystemconfig/original/templates/kubelet/_kubelet.flags#L22-L26), so let's move the labels here.